CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## dev
 
+- Use `Iterable[metadata.Distribution]` instead of `List[metadata.Distribution]` in `venv_inspect.py` (#629)
```

Comment:
We don't need a changelog here since this change has no effect on end users.

Reply:
Thank you for the suggestion @dukecat0 , the changelog entry has been removed in the last commit :+1: